build-tools/packages/build-infrastructure/vitest.config.ts

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+	test: {
+		include: ["src/test/*.vitest.?(c|m)[jt]s?(x)"],
```

Comment:
I was a bit surprised to see this way to specify the "0 or 1 time" before the group, which I recently saw in ADO syntax and thought it was a quirk specific to ADO. I see now that it's an _extended globbing_ thing, not a regex thing (thank you https://unix.stackexchange.com/a/580530). Maybe add a comment to make that explicit from the start, so people who are used to regex but not globbing don't go crazy trying to understand this?

Reply:
Do you think linking to the vitest docs is enough? https://vitest.dev/config/#include